src/partition_mesh.py

```diff
@@ -163,7 +163,10 @@ def partition_metis(mesh: Mesh, numparts: int):
         cell = mesh.cells[i]
         cellData += list(cell)
         cellPtr.append(cellPtr[-1] + len(cell))
-    libmetis = cdll.LoadLibrary(os.path.abspath("libmetisAPI.so"))
+    print(os.path.dirname(__file__))
+    print("libmetisAPI.so")
+    print(os.path.join(os.path.dirname(__file__),"libmetisAPI.so"))
+    libmetis = cdll.LoadLibrary(os.path.join(os.path.dirname(__file__),"libmetisAPI.so"))
```

Comment:
If we add the install target, then this won't work since the shared library will be installed into the `lib` directory and the python scripts into the `bin` directory, right?

Reply:
How about as follows it should work both installed and compiled case?

```python
binpath=os.path.dirname(__file__)
libpath= os.path.normpath(os.path.join(binpath+"../lib"))
if os.path.isfile(os.path.join(binpath,"libmetisAPI.so")):
       libmetispath=os.path.join(binpath,"libmetisAPI.so")
elif os.path.isfile(os.path.join(libpath,"libmetisAPI.so")):
       libmetispath=os.path.join(libpath,"libmetisAPI.so")
else:
    raise Exception("libmetisAPI.so cannot found!")
libmetis = cdll.LoadLibrary(libmetispath)
```